config/jobs/test-infra-hosts/jenkins/LinuxHostBuild.Jenkinsfile

```diff
@@ -0,0 +1,234 @@
+                            --name ${VM_NAME} \
+                            --command-id RunShellScript \
+                            --scripts 'ansible-playbook /home/jenkins/test-infra/scripts/ansible/oe-contributors-acc-setup.yml'
+
+                        sleep 15s
+                        '''
+                    )  
+                }
+            }
+        }
+
+        stage('Save VM State') {
+            steps{
+                script{
+                    sh(
+                        script: '''
+                        az vm deallocate \
+                            --resource-group ${VM_RESOURCE_GROUP} \
+                            --name ${VM_NAME}
+
+                        az vm generalize \
+                            --resource-group ${VM_RESOURCE_GROUP} \
+                            --name ${VM_NAME}
+                        
+                        img_id=\$(az image create \
+                            --resource-group ${VM_RESOURCE_GROUP} \
+                            --name myImage \
+                            --source ${VM_NAME} \
+                            --hyper-v-generation V2 | jq -r '.id')
+
+                        az sig image-definition create \
+                            --resource-group ACC-Images \
+                            --gallery-name ACC_Images \
+                            --gallery-image-definition ${GALLERY_DEFN} \
+                            --publisher ${PUBLISHER} \
+                            --offer ${OFFER} \
+                            --sku ${SKU} \
+                            --os-type Linux \
+                            --os-state generalized \
+                            --hyper-v-generation V2 || true
+
+                        YY=$(date +%Y)
+                        DD=$(date +%d)
+                        MM=$(date +%m)
+
+                        GALLERY_IMAGE_VERSION="$YY.$MM.$DD${BUILD_ID}"
+                        GALLERY_NAME="ACC_Images"
+
+                        az sig image-version delete \
+                            --resource-group "ACC-Images" \
+                            --gallery-name ${GALLERY_NAME} \
+                            --gallery-image-definition ACC-${LINUX_VERSION} \
+                            --gallery-image-version ${GALLERY_IMAGE_VERSION}
+
+                        az sig image-version create \
+                            --resource-group "ACC-Images" \
+                            --gallery-name ${GALLERY_NAME} \
+                            --gallery-image-definition ACC-${LINUX_VERSION} \
+                            --gallery-image-version "${GALLERY_IMAGE_VERSION}" \
+                            --target-regions "uksouth" "eastus2" "eastus" "westus2" "westeurope" \
+                            --replica-count 1 \
+                            --managed-image $img_id \
+                            --end-of-life-date "$(($YY+1))-$MM-$DD" \
+                            --no-wait
```

Comment:
If you don't wait for a response, does it go to the next stage right away (e.g. Tests)?

Reply:
Yup.

We need to think about how to test here. The above replicates worldwide, but we should test on the generalized image in the prior step by launching a VM then using an agreed upon testing strategy... that is not an echo statement